services/github/github-common-fetch.js

```diff
@@ -45,10 +45,11 @@ async function fetchRepoContent(
       throw new InvalidResponse({ prettyMessage: 'undecodable content' })
     }
   } else {
-    return serviceInstance._request({
+    const response = await serviceInstance._request({
       url: `https://raw.githubusercontent.com/${user}/${repo}/${branch}/${filename}`,
       errorMessages,
     })
+    return response.buffer
```

Comment:
```suggestion
    return buffer
```

Reply:
Sorry, instead of commiting your suggestion I pushed a new commit 😄